frontend/src/metabase/visualizations/components/LegendVertical.jsx

```diff
@@ -93,11 +71,7 @@ export default class LegendVertical extends Component {
       items = titles;
     }
     return (
-      <ol
-        className={cx(className, LegendS.Legend, LegendS.vertical)}
-        style={{ width: this.state.listWidth }}
```

Comment:
@kulyk Any concerns about removing this style? The legend seems to render fine without it

Reply:
https://github.com/user-attachments/assets/340525f1-34bc-4f64-a181-1b504939d47e

It was meant to prevent the legend resizing when we're hiding a particular segment and recalculating the percentages.

Reply:
Ah I missed that, thanks for the explanation. I'll update the PR to fix this